scipy-1.0/paper.tex

```diff
@@ -28,7 +28,19 @@
 \keywords{Scientific computing, Python, Mathematics}
 
 \begin{abstract}
-TODO. Abstract must be under 200 words and not include subheadings or citations.
+SciPy is an open source scientific computing library for the Python programming language.
+SciPy 1.0 was released in late 2017, about 16 years after the original
+version 0.1 release. SciPy
+has become a \emph{de facto} standard in the community, with more than 600
+unique code contributors and millions of downloads per year, despite
+the absence of an official project publication, until now. The library
```

Comment:
the "despite the absence ..." is not needed I think. that's more a letter to the editor kind of thing